controllers/admin/AdminAjaxPs_buybuttonliteController.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+* 2007-2016 PrestaShop
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+* @author    PrestaShop SA <contact@prestashop.com>
+* @copyright 2007-2015 PrestaShop SA
+* @license   http://addons.prestashop.com/en/content/12-terms-and-conditions-of-use
+* International Registered Trademark & Property of PrestaShop SA
+*/
+
+class AdminAjaxPs_buybuttonliteController extends ModuleAdminController
+{
+    public function ajaxProcessSearchProducts()
+    {
+        $context = Context::getContext();
+        $id_lang = $context->language->id;
+        $id_shop = $context->shop->id;
+        $query = pSQL(Tools::getValue('product_search'));
+
+        $sql = new DbQuery();
```

Comment:
this huge block could be moved into a `getSqlQuery()` function to improve the readability of `ajaxProcessSearchProducts()`

Reply:
Done :)